ibtk/configure

```diff
@@ -12118,16 +12118,11 @@ $as_echo "$lt_cv_ld_force_load" >&6; }
       _lt_dar_allow_undefined='$wl-undefined ${wl}suppress' ;;
     darwin1.*)
       _lt_dar_allow_undefined='$wl-flat_namespace $wl-undefined ${wl}suppress' ;;
-    darwin*) # darwin 5.x on
-      # if running on 10.5 or later, the deployment target defaults
-      # to the OS version, if on x86, and 10.4, the deployment
-      # target defaults to 10.4. Don't you love it?
-      case ${MACOSX_DEPLOYMENT_TARGET-10.0},$host in
-	10.0,*86*-darwin8*|10.0,*-darwin[91]*)
-	  _lt_dar_allow_undefined='$wl-undefined ${wl}dynamic_lookup' ;;
-	10.[012][,.]*)
+    darwin*)
+      case ${MACOSX_DEPLOYMENT_TARGET},$host in
+	10.[012],*|,*powerpc*)
 	  _lt_dar_allow_undefined='$wl-flat_namespace $wl-undefined ${wl}suppress' ;;
-	10.*)
+	*)
```

Comment:
What version of autotools did you use? This file shouldn't be changed.

Reply:
I just verified that running `bootstrap` on the branch reverses most (all?) of these change.

I could commit and squash...

Reply:
@Kaustubh-Khedkar, if you run the `bootstrap` script in the top level of the IBAMR source directory, do you see the same behavior?

IIRC, we tried to set up `bootstrap` so that it automatically installs consistent versions of `autoconf`, `automake`, and `libtool` so that users do not have to manually deal with it --- perhaps it is not working, or is not doing what we intended it to do.

Reply:
> I could commit and squash...

Boyce has a good point - there is no reason for us to get tripped up here if we can easily force push a fix. Lets focus on the test.

Reply:
> > I could commit and squash...
> 
> Boyce has a good point - there is no reason for us to get tripped up here if we can easily force push a fix. Lets focus on the test.

Sounds good, although I will hold off on doing this until the differences in the NWT test output are resolved.